client/src/components/cards/ClueOne.js

```diff
@@ -0,0 +1,80 @@
+//react
```

Comment:
I wonder if the clue card can be one component that accepts a "clueNumber" prop

Reply:
I'll look into this! I think it may be possible. I have to think about the button for user asks for hints and how that would work.